src/pow.cpp

```diff
@@ -91,10 +91,10 @@ unsigned int CalculateNextWorkRequired(arith_uint256 bnAvg,
     return bnNew.GetCompact();
 }
 
-bool CheckEquihashSolution(const CBlockHeader *pblock, const CChainParams& params)
```

Comment:
Nit: there are a few places in this commit where we just say `params` rather than saying `chainparams` or `consensusParams` explicitly.

Reply:
IIRC these were places that we ourselves introduced parameters in the past. I didn't alter the variable names to make the PR smaller, but I can push another commit that makes them consistent.

Reply:
I decided to only address this in `main.cpp`; after this PR only `Consensus::Params` is used in `pow.cpp`, so there's not much point in adding additional change lines to this PR. We can address later if desired.